util/execdetails/execdetails_test.go

```diff
@@ -101,30 +101,30 @@ func TestCopRuntimeStats(t *testing.T) {
 	stats.RecordOneCopTask(aggID, "8.8.8.8", mockExecutorExecutionSummary(3, 3, 3))
 	stats.RecordOneCopTask(aggID, "8.8.8.9", mockExecutorExecutionSummary(4, 4, 4))
 	copDetails := &CopDetails{
-		TotalKeys:                 10,
+		TotalKeys:                 15,
```

Comment:
Why this is changed?
Could you add some comments describing each field in struct `CopDetail`?

Reply:
Change this for making sure different files have different value, to validate the function doesn't typo.
Already add a comment.